LayoutTests/inspector/console/message-stack-trace-expected.txt

```diff
@@ -57,3 +58,7 @@ ASYNC CALL STACK:
 1: [N] then
 2: [F] triggerUnhandledRejectionImplicitWithException
 
+-- Running test case: Console.StackTrace.UnhandledPromiseRejection.Module
+CALL STACK:
+0: [F] (anonymous function)
```

Comment:
just to help clarify, without these changes this would look something like
```
CALL STACK:
0: [F] (anonymous function)
ASYNC CALL STACK:
0: --- enqueueJob ---
```